lib/specifications/Specification.js

```diff
@@ -110,52 +110,69 @@ class Specification {
 
 	/* === Attributes === */
 	/**
-	* @public
-	*/
+	 * Get the project name
+	 *
+	 * @public
+	 * @returns {string} Project name
+	 */
 	getName() {
 		return this._name;
 	}
 
 	/**
-	* @public
-	*/
+	 * Get the project kind
+	 *
+	 * @public
+	 * @returns {string} Project kind
```

Comment:
It's confusing to have both a 'kind' and 'type' of project. Can this be remedied?

Reply:
Thank you for pointing this out. I did a mistake by referring to a specification as a "project".

In fact, a specification can be either a "project", **or** an "extension" (this is defined by the *kind* property). There might be other kinds in the future too...

I updated the documentation in the latest commit: [`88375b7` (#514)](https://github.com/SAP/ui5-project/pull/514/commits/88375b78c2bcecb9192903a98febe1c4084609a4)

Reply:
I agree it is confusing. But sadly we introduced the additional "kind" term over two years ago when we added support for extensions. So I think it is to late to change any of the two terms now.

Reply:
That's really unfortunate. I think it would make sense to at least explain these terms better. Maybe we could put a more detailed example in, together with the definitions - what do you think?